src/Toolkit/Toolkit.UWP/UI/Controls/SearchView/SearchView.Theme.xaml

```diff
@@ -0,0 +1,662 @@
+                        Text="{Binding SearchViewModel.ActivePlaceholder, Mode=OneWay}"
+                        Visibility="{Binding ElementName=QueryEntry, Path=Text, Converter={StaticResource PlaceholderVisibilityConverter}}" />
+                    <Button
+                        Grid.Column="2"
+                        BorderThickness="0"
+                        Command="{Binding ClearCommand, Mode=OneTime}"
+                        Style="{StaticResource BarActionButtonStyle}"
+                        ToolTipService.ToolTip="{TemplateBinding ClearSearchTooltipText}"
+                        Visibility="{Binding ElementName=QueryEntry, Path=Text, Converter={StaticResource PlaceholderVisibilityConverter}, ConverterParameter='NotEmpty'}">
+                        <Path Data="M3.98 11.303L7.281 8 3.98 4.697l.707-.707L7.99 7.293l.01-.01.01.01 3.304-3.303.707.707L8.718 8l3.303 3.303-.707.707L8.01 8.707l-.01.01-.01-.01-3.304 3.303z" Style="{StaticResource ActionButtonPathStyle}" />
+                    </Button>
+                    <Button
+                        Grid.Column="3"
+                        BorderThickness="1,0,0,0"
+                        Command="{Binding SearchCommand, Mode=OneTime}"
+                        Style="{StaticResource BarActionButtonStyle}"
+                        ToolTipService.ToolTip="{TemplateBinding SearchTooltipText}">
+                        <Path Data="M13.936 13.24L9.708 9.01a4.8 4.8 0 1 0-.69.69l4.228 4.228a.488.488 0 0 0 .69-.69zM6.002 9.8A3.8 3.8 0 1 1 8.69 8.686a3.778 3.778 0 0 1-2.687 1.112z" Style="{StaticResource ActionButtonPathStyle}" />
+                    </Button>
+                    <ProgressBar
+                        Grid.Row="0"
+                        Grid.ColumnSpan="4"
+                        Style="{StaticResource ProgressIndicatorStyle}"
+                        Visibility="{Binding SearchViewModel.IsWaiting, Mode=OneWay}" />
+                </Grid>
+            </Border>
+            <Popup IsOpen="{Binding SearchViewModel.Suggestions.Count, Mode=OneWay, Converter={StaticResource CollectionIsEmptyToBoolConverter}, ConverterParameter='NotEmpty'}" VerticalOffset="34">
+                <Popup.Resources>
+                    <CollectionViewSource
+                        x:Key="GroupedSuggestionsEx"
+                        IsSourceGrouped="True"
+                        Source="{Binding GroupedSuggestions, Mode=OneWay}" />
+                </Popup.Resources>
+                <Border
+                    Width="{TemplateBinding Width}"
+                    BorderThickness="1,0,1,1"
+                    Style="{StaticResource ResultAreaBorderStyle}"
+                    Visibility="{Binding SearchViewModel.Results.Count, Mode=OneWay, Converter={StaticResource CollectionIsEmptyToBoolConverter}, ConverterParameter='Empty'}">
+                    <Grid>
+                        <ListView
+                            ItemTemplate="{StaticResource SuggestionTemplate}"
```

Comment:
I was conflicted about how many customization properties it made sense to provide. It would be trivial to add properties for `SuggestionTemplate`, `ResultTemplate`, `SourceTemplate`, etc, but then would it also make sense to `SuggestionListStyle`, `SuggestionListItemContainerStyle`, and so on? The UI is complicated enough that it may not make sense to change part without changing the whole thing.